RecoLocalTracker/SiPixelRecHits/src/SiPixelTemplateSplit.cc

```diff
@@ -103,7 +103,7 @@ int SiPixelTemplateSplit::PixelTempSplit(int id, float cotalpha, float cotbeta,
 {
     // Local variables 
 	int i, j, k, binq, midpix, fypix, nypix, lypix, logypx, lparm;
-	int fxpix, nxpix, lxpix, logxpx, shifty, shiftx, nyzero[TYSIZE];
+	int fxpix, nxpix, lxpix, logxpx, shifty, shiftx, nyzero[TYSIZE]{};
```

Comment:
I initialized it here rather than on line 226 because in line 211, all values of the array get initialized to zero unless it never goes into that for loo. Sorry what do you mean by old initialization is still in place?